man/jail.conf.5

```diff
@@ -146,6 +146,12 @@ Ensure syslog or the program that generates the log file isn't configured to com
 .B logencoding
 encoding of log files used for decoding. Default value of "auto" uses current system locale.
 .TP
+.B banaction
+default banning action (iptables-multiport) for all jails specified in the \fI[DEFAULT]\fR section.
```

Comment:
It could also be specified per jail. Again, DEFAULTS are just defaults. So no need to state where it is since it can be anywhere

On October 31, 2015 10:18:58 AM EDT, "Serg G. Brester" <notifications@github.com> wrote:
>> @@ -146,6 +146,12 @@ Ensure syslog or the program that generates the
>log file isn't configured to com
>>  .B logencoding
>>  encoding of log files used for decoding. Default value of "auto"
>uses current system locale.
>>  .TP
>> +.B banaction
>> +default banning action (iptables-multiport) for all jails specified
>in the \fI[DEFAULT]\fR section.
>
>Yes, but I meant: the banning action (for all jails) specified in the
>`[DEFAULT]` section.
>Not the jails in default section. My bad english :(
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/fail2ban/fail2ban/pull/1238/files#r43572222